handsontable/src/plugins/manualRowResize/manualRowResize.js

```diff
@@ -321,14 +321,14 @@ export class ManualRowResize extends BasePlugin {
   setupGuidePosition() {
     const handleWidth = parseInt(outerWidth(this.#handle), 10);
     const handleEndPosition = parseInt(this.#handle.style[this.inlineDir], 10) + handleWidth;
-    const maximumVisibleElementWidth = parseInt(this.hot.view.maximumVisibleElementWidth(0), 10);
+    const tableWidth = parseInt(this.hot.view.getTableWidth(), 10);
```

Comment:
AFAIK the method already returns a number, there is no need to cast a value.
```suggestion
    const tableWidth = this.hot.view.getTableWidth();
```